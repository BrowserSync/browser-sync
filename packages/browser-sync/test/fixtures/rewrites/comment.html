<!DOCTYPE html>
<html>
    <body>
        <p>This is an experiment.dev</p>
        <p><img src="/images/experiment.dev/example.jpg" /></p>
        <!-- This experiment.dev should prove interesting -->
        <p><a href="http://experiment.dev/example.html">Link</a></p>
    </body>
</html>
