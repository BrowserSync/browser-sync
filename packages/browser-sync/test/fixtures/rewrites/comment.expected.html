<!doctype html>
<html><body>
<p>This is an experiment.dev</p>
<p><img src="/images/experiment.dev/example.jpg"></p>
<!-- This experiment.dev should prove interesting -->
<p><a href="http://192.168.0.4:3002/example.html">Link</a></p>
</body></html>
